DuckDuckGo/TabsModel.swift

```diff
@@ -75,7 +75,7 @@ public class TabsModel: NSObject, NSCoding {
 
     var currentTab: Tab? {
         let index = currentIndex
-        return tabs[index]
+        return tabs.indices.contains(index) ? tabs[index] : nil
```

Comment:
There's also an extension with `safe` subscript for `RandomAccessCollection` in BSK, but that will work just fine.